boot/efi/multiboot.c

```diff
@@ -7,6 +7,7 @@
 
 static multiboot_memory_map_t *multiboot_mmap;
 static UINT32 mmap_elems_count;
+static tilck_extra_boot_info *extra_boot_info;
```

Comment:
Why a pointer? You'd need to make it point to some valid mem area. That would require an allocation etc.
Just make it a plan object, as you'll going to make the actual allocation later anyways, and copy this object there.